docs/guides/Orders/orders-feed.md

```diff
@@ -402,24 +363,24 @@ Below is an example of a hook notification request body made to the integration
 [block:callout]
 {
   "type": "warning",
-  "body": "When using the Hook, it is important to be aware that it is a reactive feature. This means your middleware or ERP system must be ready to deal with whatever volume of data is sent by the Hook. Large peaks in sales - due to Black Friday, for example - tend to generate increase in Hook notifications. If the implementation is not prepared for this peak, it may cause the integration to malfunction, compromising the store’s ability to handle orders and receive further notifications. Learn more about how to deal with this issue in the next session."
+  "body": "When using a hook, it's important to be aware that it's a reactive feature. This means your middleware or ERP system must be ready to deal with whatever volume of data the hook sends. Large peaks in sales – due to Black Friday, for example – tend to increase hook notifications. If the implementation is not prepared for this peak, it may cause problems in the integration, compromising the store’s ability to handle orders and receive further notifications. Learn more about how to deal with this issue in the next section."
 }
 [/block]
```

Comment:
```suggestion
>⚠️ When using a hook, it's important to be aware that it's a reactive feature. This means your middleware or ERP system must be ready to deal with whatever volume of data the hook sends. Large peaks in sales – due to Black Friday, for example – tend to increase hook notifications. If the implementation is not prepared for this peak, it may cause problems in the integration, compromising the store’s ability to handle orders and receive further notifications. Learn more about how to deal with this issue in the next section.
```